src/pages/pomoclock/PomoClock.jsx

```diff
@@ -0,0 +1,194 @@
+
+		setInfoMessage("Clock Started");
+		clearMessage();
+		clockHandler("isStarted", true);
+
+		intervalTime.current = setInterval(() => {
+			if (secondsTime.current === 0) {
+				clearInterval(intervalTime);
+				switchMode();
+			} else {
+				secondsTime.current--;
+				setSeconds(secondsTime.current);
+			}
+		}, 1000);
+	};
+
+	const pauseHandler = () => {
+		if (clockState.isPaused) {
+			handleStart();
+			clockHandler("isPaused", false);
+			setInfoMessage("Clock Resumed");
+		} else {
+			clearInterval(intervalTime.current);
+			clockHandler("isPaused", true);
+			setInfoMessage("Clock Paused");
+		}
+
+		clearMessage();
+	};
+
+	const restartHandler = () => {
+		setSeconds(time * 60);
+		secondsTime.current = time * 60;
+		clearInterval(intervalTime.current);
+		setInfoMessage("Clock reset");
+		setClockState(clockInitalState);
+		setPomoMode("focus");
+		clearMessage();
+	};
+
+	useEffect(() => {
+		setSeconds(time * 60);
+		secondsTime.current = time * 60;
+	}, [time]);
+
+	const totalSeconds = pomoMode === "focus" ? time * 60 : 5 * 60;
+	const secondsLeft = seconds % 60;
+	const minutesLeft = Math.floor(seconds / 60);
+	const percentValue = (seconds / totalSeconds) * 100;
+
+	return (
+		<div>
+			<main className="pomoclock-container my-44 mx-auto">
+				<section className="pomoclock">
+					<div className="pomoclock-bar my-4 mx-auto">
+						<CircularProgressbar
+							value={percentValue}
+							text={`${minutesLeft}m:${secondsLeft}s`}
+							counterClockwise={true}
+							styles={{
+								text: {
+									fontSize: "1rem",
+								},
+								path: {
+									stroke:
+										pomoMode === "focus"
+											? "hsl(242, 51%, 61%)"
+											: "hsl(69, 58%, 89%)",
+								},
+								trail: {
+									stroke:
+										pomoMode === "focus"
+											? "hsl(69, 58%, 89%)"
+											: "hsl(242, 51%, 61%)",
+								},
+							}}
+						/>
+					</div>
+
+					<div className="clock-btn grid my-8 mx-auto gap-4 text-center">
+						<button
+							className="btn btn-primary"
+							onClick={handleStart}
+							disabled={clockState.isStarted}
+						>
+							<span className="flex align-center justify-center">
+								<FaPlay className="mt-2" /> Start
+							</span>
+						</button>
+
+						<button
+							className="btn btn-success"
+							onClick={pauseHandler}
+							disabled={!clockState.isStarted}
+						>
+							{clockState.isPaused ? (
+								<span className="flex align-center justify-center">
+									<FaPause className="mt-2" /> Resume
+								</span>
+							) : (
+								<span className="flex align-center justify-center">
+									<FaStop className="mt-2" /> Pause
+								</span>
+							)}
+						</button>
+
+						<button className="btn btn-info" onClick={restartHandler}>
+							Reset
+						</button>
+					</div>
+
+					{infoMessage && (
+						<span className="info-message text-center text-3xl absolute p-4 rounded-md">
+							{infoMessage}
+						</span>
```

Comment:
- UI Looks great
- Code is easy to understand
- working as expected
- Ready to merge